task2/time.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+db={1:("одна минута", "второго", "одной", "первый", "два"),2:("двe минуты", "третьего", "двух", "второй", "три"),3:("три минуты", "четвертого", "трех", "третий", "четыре"),
+4:("четыре минуты", "пятого", "четырех", "четвертый", "пять"),5:("пять минут", "шестого", "пяти", "пятый", "шесть"),6:("шесть минут", "седьмого", "шести", "шестой", "семь"),7:("семь минут", "восьмого", "семи", "седьмой", "восемь"),8:("восемь минут", "девятого", "восьми", "восьмой", "девять"),9:("девять минут", "десятого", "девяти", "девятый", "десять"),
+10:("десять минут", "одинадцатого", "десяти", "десятый", "одинацать"),11:("одиннадцать минут", "двенадцатого", "одинадцати", "одинадцатый", "двенадцать"),12:("двенадцать минут", "первого", "двенадцати", "двенадцатый", "час"),
+13:("тринадцать минут", "второго", "тринадцати", "тринадцатый", "два"),14:("четырнадцать минут", "третьего", "четырнадцати", "четырнадцатый", "три"),15:("пятнадцать минут", "четвертого", "пятнадцати", "пятнадцатый", "четыре"),
+16:("шестнадцать минут", "пятого", "шестнадцатый", "", "пять"),17:("семнадцать минут", "шестого", "", "семнадцатый", "шесть"),18:("восемнадцать минут", "седьмого", "", "восемнадцатый", "семь"),
+19:("девятнадцать минут", "восьмого", "", "девятнадцатый", "восемь"),20:("двадцать минут", "девятого", "", "двадцатый", "девять"),21:("двадцать одина минута", "десятого", "", "двадцать первый", "десять"),
+22:("двадцать две минуты", "одиннадцатого", "", "двадцать второй", "одинадцать"),23:("двадцать три минуты", "двенадцатого", "", "двадцать третий", "двенадцать"),24:("двадцать четыре минуты", "первого", "", "двадцать четвертый", "час"),25:("двадцать пять минут"),26:("двадцать шесть минут"),
+27:("двадцать семь минут"),28:("двадцать восемь минут"),29:("двадцать девять минут"),30:("тридцать"),31:("тридцать одна минута"),32:"тридцать две минуты",33:"тридцать три минуты",34:"тридцать четыре минуты",35:"тридцать пять минут",36:"тридцать шесть минут",
+37:"тридцать семь минут",38:"тридцать восемь минут",39:"тридцать девять минут",40:"сорок минут",41:"сорок одна минута",42:"сорок две минуты",43:"сорок три минуты",44:"сорок четырые минуты",45:"сорок пять ",46:"сорок шесть",
+47:"сорок семь",48:"сорок восемь ",49:"сорок девять ",50:"пятьдесят",51:"пятьдесят один",52:"пятьдесят два",53:"пятьдесят три",54:"пятьдесят четыре ",55:"пятьдесят пять",
+56:"пятьдесят шесть",57:"пятьдесят семь",58:"пятьдесят восемь",59:"пятьдесят девять", 0:("двадцать четыре минуты", "первого", "", "двадцать четвертый", "час")}
+x=datetime.datetime.now()
+min=datetime.datetime.today().minute
+hour=datetime.datetime.today().hour
+choise=input("Введите Yes , если хотите ввести время или NO,если время текущее ")
+if choise.lower()=="yes":
+    person_time=datetime.datetime.strptime(input('Time: '), "%H:%M")
```

Comment:
you could provide a user with some tip about input format (and then validate the input for the greater good)